src/transaction.js

```diff
@@ -395,8 +396,12 @@ Transaction.prototype.supportsJoinSplits = function () {
   return coins.isZcash(this.network) && this.version >= Transaction.ZCASH_JOINSPLITS_SUPPORT_VERSION
 }
 
+Transaction.prototype.supportsDashSpecialTransactions = function () {
```

Comment:
`supports` on `Transaction` seems confusing. Tx is just is, so it is either special Tx, or not. I'm not sure if having this function as standalone is any useful, so maybe this should just be included in `isDashSpecialTransaction`. Alternatively, maybe `versionSupportsDashSpecialTransactions` or something...

Reply:
Renamed to `versionSupportsDashSpecialTransactions`.